tests/integration/data_sources_and_expectations/test_canonical_expectations.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+
+import great_expectations.expectations as gxe
+from tests.integration.conftest import parameterize_batch_for_data_sources
+from tests.integration.test_utils.data_source_config import (
+    PandasDataFrameDatasourceConfig,
+)
+
+
+@parameterize_batch_for_data_sources(
+    data_source_configs=[
+        PandasDataFrameDatasourceConfig(),
+    ],
+    data=pd.DataFrame({"a": [1, 2]}),
+)
+def test_min(batch_for_datasource) -> None:
```

Comment:
super nit, but since we're setting the pattern - why not simplify param to just `batch`?

Reply:
My main argument in terms of making it something less generic that just `batch` is that this is a bit of magic, and I wanted the name to be unique enough to be easily searchable. Maybe that isn't compelling, though, idk. what do you think? fwiw go-to-definition works just fine with this in my editor, so it should still be discoverable if we rename it to something like `batch`